### PR TITLE
C cheatsheet: Added pointers to c cheatsheet

### PR DESCRIPTION
**Pull request by preemeijer**
If this is something else:
C cheatsheet: Added pointers to c cheatsheet

## People to notify

@harisphnx 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/c_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/c.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
         "Other Operators",
         "Conversion Characters",
         "Escape Sequences",
-        "Order of Precedence"
+        "Order of Precedence",
+	"Pointers"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/c.json]: There's a hard-tab here, I'd be murdered if I merged with that in there

File: share/goodie/cheat_sheets/json/c.json [hunk 2]
```diff
@@ -281,6 +282,24 @@
                 "val": "Comma - associativity goes from left to right",
                 "key": "[ , ]"
             }
-        ]
+        ],
+	"Pointers": [
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/c.json]: And here!

File: share/goodie/cheat_sheets/json/c.json [hunk 3]
```diff
@@ -281,6 +282,24 @@
...
+            },
+            {
+                "val": "Declaring a pointer to a variable",
+                "key": "*"
+            },
+            {
+                "val": "Declaring a void pointer",
+                "key": "void *"
+            }
+	]
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/c.json]: And here!

**Comment by mintsoft** [comment]
@preemeijer Yo paul, sorry this took so long to get around to. If you tidy up the tabs I'll merge asap

**Comment by preemeijer** [comment]
@mintsoft Really no problem. I really hate that I did not deliver a clean merge request.

**Comment by mintsoft** [comment]
Cheers @preemeijer

**Pull request merged**